Tests/PersistenceHandler/RabbitMqPersistenceHandlerTest.php

```diff
@@ -290,10 +314,19 @@ public function testPersistRetrieveValidDataFromPdfFile()
 
         $persistenceHandler = new RabbitMqPersistenceHandler($queue, $pdfParser);
 
-        $this->assertNull($persistenceHandler->persist($resource));
+        $crawlJob = $this
+            ->getMockBuilder('Simgroep\ConcurrentSpiderBundle\CrawlJob')
+            ->setMethods(null)
+            ->disableOriginalConstructor()
+            ->getMock();
+
+        $this->assertNull($persistenceHandler->persist($resource, $crawlJob));
     }
 
-    public function testPersistRetrieveValidDataFromWebPageWithAllDefaultValuesInFields()
+    /**
+     * @test
+     */
+    public function persistRetrieveValidDataFromWebPageWithAllDefaultValuesInFields()
```

Comment:
Well I like the @test annotation which was hinted by Peter. It gives you more space to write down a descriptive test method name. For internal method you should use the private or at least the protected modifiers which will make it clear that it is internally used.